### ISSUE TITLE
Menu not hidden on startup

### ISSUE THREAD

**Issue by orschiro** [pcorey/sublime-hide-menu#2]
Hi @pcorey,

Many thanks for this plugin!

I installed it on my Arch Linux machine. 

Installed version of Sublime is `3.3065-1`.

I launch sublime in a folder from the command line as follows:

```
conkytimer - master ❯ subl3 .
```

However, the menu is not hidden.

Any idea why?

Thanks for your advice!

Robert

**Comment by pcorey** [comment]
Hey! Do you have another instance of Sublime running? I've noticed that running "subl ." while another instance is running will cause the new window to open seemingly without running the Hide Menu plugin. That's definitely and issue I'll try to look into.

You may also be experiencing issue #1.

It's a shame such a seemingly simple plugin has such major issues. Sorry about that...

**Comment by orschiro** [comment]
> Hey! Do you have another instance of Sublime running? I've noticed that running "subl ." while another instance is running will cause the new window to open seemingly without running the Hide Menu plugin. That's definitely and issue I'll try to look into.

Yes, this was indeed the case! Thanks for looking into it. In that sense it is related to #1.

The first window has the menu hidden. Thus the secondly opened one is toggling state and showing the menu again.

I hope there is a solution to this, though I don't know how...
